actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -130,12 +130,21 @@ def truncate(text, options = {}, &block)
         end
       end
 
-      # Highlights one or more +phrases+ everywhere in +text+ by inserting it into
-      # a <tt>:highlighter</tt> string. The highlighter can be specialized by passing <tt>:highlighter</tt>
-      # as a single-quoted string with <tt>\1</tt> where the phrase is to be inserted (defaults to
-      # <tt>"<mark>\1</mark>"</tt>) or passing a block that receives each matched term. By default +text+
-      # is sanitized to prevent possible XSS attacks. If the input is trustworthy, passing false
-      # for <tt>:sanitize</tt> will turn sanitizing off.
+      # Highlights one or more phrases everywhere in the given +text+ by inserting them into
+      # a custom <tt>:highlighter</tt> string. The highlighter can be specialized by passing <tt>:highlighter</tt>
+      # as a single-quoted string with <tt>\1</tt> where the phrase is to be inserted or passing a block
+      # that receives each matched term. By default, the +text+ is sanitized to prevent possible XSS attacks.
```

Comment:
```suggestion
      # Highlights occurrences of +phrases+ in +text+ by formatting them with a
      # highlighter string. +phrases+ can be one or more strings or regular
      # expressions. The result will be marked HTML safe. By default, +text+ is
      # sanitized before highlighting to prevent possible XSS attacks.
      #
      # If a block is specified, it will be used instead of the highlighter
      # string. Each occurrence of a phrase will be passed to the block, and its
      # return value will be inserted into the final result.
```